src/MCPClient/lib/clientScripts/post_store_aip_hook.py

```diff
@@ -164,6 +195,16 @@ def post_store_hook(job, sip_uuid):
     # POST-STORE CALLBACK
     storage_service.post_store_aip_callback(sip_uuid)
 
+    # When not using SIP arrangement, we perform best-effort deletion of the
+    # original transfer directory under currentlyProcessing.
+    if not transfer_uuids:
+        try:
+            delete_transfer_directory(job, sip_uuid)
+        except Exception as err:
+            job.pyprint("Failed to delete transfer directory: ", err, file=sys.stderr)
+            return
+        job.pyprint("Transfer directory deleted.")
```

Comment:
Do you think it'd be useful to log what `transfer_path` was deleted?

Reply:
Done!